### PR TITLE
[Improve][Connector-V2] RocketMQ Sink add message tag config

### PR DESCRIPTION
**Pull request by wengys**
### Purpose of this pull request

Add static message tag support for RocketMQ sink

### Check list

* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  2. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
  3. Add ci label in [label-scope-conf](https://github.com/apache/seatunnel/blob/dev/.github/workflows/labeler/label-scope-conf.yml)
  4. Add e2e testcase in [seatunnel-e2e](https://github.com/apache/seatunnel/tree/dev/seatunnel-e2e/seatunnel-connector-v2-e2e/)
  5. Update connector [plugin_config](https://github.com/apache/seatunnel/blob/dev/config/plugin_config)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by Hisoka-X** [comment]
Could you add test case for this feature?

**Comment by wengys** [comment]
> Could you add test case for this feature?

Sure, done. However I have no idea why test case testSinkRocketMq failed on Spark3 environment. Would you please give me a hand figure out why these test case failed?

File: seatunnel-e2e/seatunnel-connector-v2-e2e/connector-rocketmq-e2e/src/test/resources/rocketmq-sink_fake_to_rocketmq_message_tag.conf [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  }
+}
+
+transform {
+}
+
+sink {
+  Rocketmq {
+    name.srv.addr = "rocketmq-e2e:9876"
+    topic = "test_topic"
```
Comment (Hisoka-X) [inline seatunnel-e2e/seatunnel-connector-v2-e2e/connector-rocketmq-e2e/src/test/resources/rocketmq-sink_fake_to_rocketmq_message_tag.conf]: I think you should use another topic, this topic be reused will let other job check data count not right.

Comment (wengys) [reply seatunnel-e2e/seatunnel-connector-v2-e2e/connector-rocketmq-e2e/src/test/resources/rocketmq-sink_fake_to_rocketmq_message_tag.conf]: Thanks, tests now passed

File: seatunnel-connectors-v2/connector-rocketmq/src/main/java/org/apache/seatunnel/connectors/seatunnel/rocketmq/config/ProducerConfig.java [hunk 2]
```diff
@@ -32,6 +32,12 @@ public class ProducerConfig extends Config {
                     .noDefaultValue()
                     .withDescription("RocketMq topic name. ");
 
+    public static final Option<String> TAG =
+            Options.key("tag")
+                    .stringType()
+                    .noDefaultValue()
+                    .withDescription("RocketMq message tag. ");
```
Comment (Hisoka-X) [inline seatunnel-connectors-v2/connector-rocketmq/src/main/java/org/apache/seatunnel/connectors/seatunnel/rocketmq/config/ProducerConfig.java]:
```suggestion
                    .withDescription("RocketMq message tag.");
```

**Review by Hisoka-X — APPROVED** [review]
LGTM. Thanks @wengys !

**Review by hailin0 — APPROVED** [review]

**Pull request merged**